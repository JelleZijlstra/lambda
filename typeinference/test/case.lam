let x = inl 3 in
let y = inr 3 in
let f = \z. case z of (\x. print 2) | (\x. print 3) in
(f x;
f y)
