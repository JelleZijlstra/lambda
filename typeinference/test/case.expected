%%stdout%%
2
3
Result: module
	let __result__ = ()
end
%%stderr%%
